samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb

```diff
@@ -22,6 +22,37 @@
     "## Setup"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Set your output and project. !!!Must Do before you can proceed!!!\n",
+    "EXPERIMENT_NAME = 'demo'\n",
+    "OUTPUT_DIR = 'Your-Gcs-Path' # Such as gs://bucket/objact/path\n",
+    "PROJECT_NAME = 'Your-Gcp-Project-Name'\n",
+    "BASE_IMAGE='gcr.io/%s/pusherbase:dev' % PROJECT_NAME\n",
+    "TARGET_IMAGE='gcr.io/%s/pusher:dev' % PROJECT_NAME\n",
+    "KFP_PACKAGE = 'https://storage.googleapis.com/ml-pipeline/release/0.1.3-rc.2/kfp.tar.gz'\n",
+    "TRAIN_DATA = 'gs://ml-pipeline-playground/tfx/taxi-cab-classification/train.csv'\n",
+    "EVAL_DATA = 'gs://ml-pipeline-playground/tfx/taxi-cab-classification/eval.csv'\n",
+    "HIDDEN_LAYER_SIZE = '1500'\n",
+    "STEPS = 3000\n",
+    "DATAFLOW_TFDV_IMAGE = 'gcr.io/ml-pipeline/ml-pipeline-dataflow-tfdv:0.1.3-rc.2'#TODO-release: update the release tag for the next release\n",
```

Comment:
Taking the images out of the components does not look great.
Can we do the following: The notebook compiles the pipeline as normal, but then the image buckets and tags are replaced. Similarly to how the sample tests are performed.